src/lib/network/wicked.rb

```diff
@@ -16,27 +16,5 @@ def reload_config(devs)
 
       SCR.Execute(BASH_PATH, "wicked ifreload #{devs.join(" ")}").zero?
     end
-
-    # Does an ifdown for each device named in devs
-    #
-    # @param [Array] list of device names
-    # @return [Boolean] true if devices bring down; false otherwise
-    def bring_down(devs)
```

Comment:
It makes sense to remove both methods if we are not using them, and in the future will see. Also don't know exactly if there are some benefits between called it with multiple interfaces or calling it once per interface.